src/mumble/GlobalShortcut.cpp

```diff
@@ -625,6 +625,27 @@ void GlobalShortcutConfig::on_qpbRemove_clicked(bool) {
 	qlShortcuts.removeAt(idx);
 }
 
+void GlobalShortcutConfig::on_qpbImport_clicked() {
+	commit();
+	QString filePath = QFileDialog::getOpenFileName(this, tr("Import shortcuts"), QString(), QLatin1String("*.ini"));
```

Comment:
Imho should warn before overwriting all existing shortcuts. We can't expect the user to have read the tooltip and we don't have an undo button. Doesn't necessarily have to be a separate dialog box. Maybe put it in the selector title?

Reply:
I think the title would not be enough. It would have to be an adequately named action button, which is not feasible with standard forms.
Warning in a dialog box seems better.